### ISSUE TITLE
Unable to access zoom after upgrading proxyman

### ISSUE THREAD

**Issue by MichaelGofron** [ProxymanApp/Proxyman#2089]
## Description
Unable to access zoom after upgrading proxyman

## Steps to Reproduce

1. Try using zoom call
2. See error message saying cannot connect

I tried both in the zoom app and on chrome through google calendar with attached zoom: 

Chrome:
<img width="713" alt="Screenshot 2024-07-17 at 10 34 44" src="https://github.com/user-attachments/assets/97356f64-dcb5-4f43-a606-9b6237330d11">

zoom app:
<img width="480" alt="Screenshot 2024-07-17 at 07 37 02" src="https://github.com/user-attachments/assets/c4115a53-594f-48d4-b309-5e8e41c4af12">

Is there a way I can download an old version of proxyman to use instead? 

## Current Behavior

Can no longer access zoom with the latest behavior

## Expected Behavior

Should be able to use zoom

## Environment

- App version: 5.6.1
- macOS version: macOS Sonoma 14.5

**Comment by NghiaTranUIT** [comment]
@MichaelGofron you should remove Zoom or Domains's zoom in the Tool menu -> SSL Proxying List -> Include Table

I suppose that Proxyman tries to decrypt Zoom data, but Zoom has SSL Pinning, so you get errors.

**Comment by MichaelGofron** [comment]
Hm, I don't have any links to zoom in the Include Table currently. Just links to the app that I'm working on

**Comment by NghiaTranUIT** [comment]
You can download old build at https://github.com/ProxymanApp/Proxyman/releases

If there is no Zoom the SSL Proxying List, Proxyman doesn't decrypt its HTTPS data. If you close Proxyman, is Zoom fixed?

**Comment by MichaelGofron** [comment]
Hm, I tried 5.0.0 build but ran into the same issue. It seemed to only happen after I updated proxyman. 
If I close proxyman, it seems all my network stops working (chrome, zoom, etc.)

**Comment by NghiaTranUIT** [comment]
I don't think it's related. 

Can you check whether or not the Helper Tools is installed in the Proxyman Setting -> Advance -> Helper Tool.

If it's installed, when you close the app, Proxyman will revert the Proxy to normal state, and you can access the Internet.

**Comment by MichaelGofron** [comment]
Hm yeah it seems enabled:


<img width="601" alt="Screenshot 2024-07-17 at 10 56 45" src="https://github.com/user-attachments/assets/ec87d8eb-1230-40ad-9abd-c69c651d8949">

**Comment by NghiaTranUIT** [comment]
I've tested with the latest Proxyman + Zoom. No SSL Proxying for the Zoom app and it work fine.

![Screenshot 2024-07-17 at 9 00 50 AM](https://github.com/user-attachments/assets/5f127785-bf48-4d31-a2d6-5d5548a15567)

I guess the problem is not from Proxyman.

If it doesn't fix you, please try to do Factory Reset in the Help menu -> Advance -> Factory Reset

**Comment by MichaelGofron** [comment]
I ended up restarting my computer and it seems to work now. 

I tried testing a couple conditions and one issue I had was when trying to connect with my work vpn (vendor palo alto networks) with proxyman it wouldn't open zoom. I closed both and tried again and it was able to.

**Comment by MichaelGofron** [comment]
I tried again this morning and had the same issue. It seems my corporate vpn and proxyman are interfering with each other. Some applications like zoom and youtube on chrome are not connecting when they are both enabled

<img width="1198" alt="Screenshot 2024-07-18 at 07 11 05" src="https://github.com/user-attachments/assets/92a6fd53-388b-407b-94f6-cd1b6c6876a7">

**Comment by NghiaTranUIT** [comment]
Yes, VPN, it's the point. Proxyman or other web debugging proxy, like Charles Proxy doesn't work with VPN. It causes some bug that I can't fix.

Recommend turning off any VPN before using Proxyman.
